common/src/main/java/com/google/auto/common/BasicAnnotationProcessor.java

```diff
@@ -505,74 +525,402 @@ private ImmutableSetMultimap<Class<? extends Annotation>, Element> toClassKeyedM
     }
   }
 
+  /* ********************************************************************************* */
+  /* Element blue prints ************************************************************* */
```

Comment:
This is very minor, but the English word is "blueprint" with no space or hyphen. So the various identifiers should have `blueprint` or `Blueprint` in them.

Reply:
Thank you for this.